pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -226,6 +232,21 @@ func validateSource(client kubecli.KubevirtClient, vmClone *clonev1alpha1.Virtua
 	return causes
 }
 
+func validateCloneHostNameNotConformingToDNSLabelRules(field *k8sfield.Path, cloneHostname string) (causes []metav1.StatusCause) {
+	if cloneHostname != "" {
+		errors := validation.IsDNS1123Label(cloneHostname)
```

Comment:
nit: you might be able to create a new function `func validateHostNameNotConfirmingToDNSLabelRules(field *k8sfield.Path, hostname string)` and use the same to be called here and https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go#L1522-L1529 here.

This will make sure that vmClone and VMI both dont re-use the same logic and test cases avoiding redundency

Reply:
OK, Thank you very much. Follow your advice.